src/altinn-app-frontend/src/utils/appMetadata.ts

```diff
@@ -61,18 +67,16 @@ export function getLayoutSetIdForApplication(
  * @returns the current data type
  */
 export function getCurrentDataTypeForApplication(
-  application: IApplication,
-  instance?: IInstance,
-  layoutSets?: ILayoutSets,
+  params: IGetDataTypeForApplicationParams
```

Comment:
Prefer to spread this object directly here, so we avoid using `params` prefix everywhere.
```suggestion
  { application, instance, layoutSets }: IGetDataTypeForApplicationParams
```